brushfire-finatra/build.sbt

```diff
@@ -0,0 +1,20 @@
+name := "brushfire-finatra"
+
+resolvers += Resolvers.twttr
```

Comment:
Hm, did you remove your local ivy cache before you tried to build the second time?